<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор XML файлов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_cfg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>конфигурация</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>директория с XML файлами</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_outf">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_driver">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>драйвер</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SQL сервер</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sqlserver">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>порт</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>база данных</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dbname">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>имя пользователя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>количество строк в одном XML файле</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_batch_size">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save_cfg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>сохранить изменения</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_table">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>таблица в базе</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_work">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>471</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>монитор обработки</string>
    </property>
    <widget class="QPushButton" name="pushButton_exec">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>запустить обработку</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>строк в базе</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_db_rows">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>текущий XML</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cur_xml">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_info">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>431</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
